src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -192,6 +226,90 @@ public function get_item_schema() {
 			'title'      => 'theme',
 			'type'       => 'object',
 			'properties' => array(
+				'author'         => array(
+					'description' => __( 'The theme author.' ),
+					'type'        => 'string',
+					'readonly'    => true,
+					'properties'  => array(
+						'raw'      => array(
+							'description' => __( 'The theme author\'s name, as found in the theme header.' ),
+							'type'        => 'string',
+							'readonly'    => true,
+						),
+						'rendered' => array(
+							'description' => __( 'HTML for the theme author, transformed for display.' ),
+							'type'        => 'string',
+							'readonly'    => true,
+						),
+					),
+				),
+				'author_uri'     => array(
+					'description' => __( 'The website of the theme author.' ),
+					'format'      => 'uri',
+					'type'        => 'string',
```

Comment:
This needs to be an object here as properties are defined.

Reply:
```suggestion
					'type'        => 'object',
```

Reply:
Yeah, great catch here. It's a `raw`, `render` like the other properties.

Reply:
Great catch indeed! :sweat_smile: